packages/wp-story-editor/src/api/story.js

```diff
@@ -167,7 +167,7 @@ export function saveStoryById(config, story) {
       url: embedded?.['wp:featuredmedia']?.[0]?.source_url || '',
     };
 
-    return data;
+    return snakeToCamelCaseObjectKeys(data);
```

Comment:
I don't think we can or should do this.

The REST API expects snake_case, but here everything is changed to camelCase

Reply:
> REST API expects snake_case

Trying to understand this better. Here we are updating the response inside `.then()`. 
The response of `saveStoryById` looks like this

```js
{
    'slug': '130',
    'status': 'publish',
    'link': 'http://localhost:8899/web-stories/130',
    'preview_link': 'http://localhost:8899/web-stories/130',
    'edit_link': 'http://localhost:8899/wp-admin/post.php?post=130&action=edit',
    'embed_post_link': 'http://localhost:8899/wp-admin/post-new.php?from-web-story=130',
    'featured_media': {
      'id': 0,
      'height': 0,
      'width': 0,
      'url': '',
    },
  }
```

which is being used here, after getting converted to camel case.

https://github.com/google/web-stories-wp/blob/c29543c002a5f6c407e0d8f56b2d878b44ae7acf/packages/story-editor/src/app/story/actions/useSaveStory.js#L83-L91

Reply:
Argh, my bad! Somehow thought this was the request data, not response data.